awx/api/serializers.py

```diff
@@ -2966,9 +2948,11 @@ def get_related(self, obj):
 
 class SystemJobSerializer(UnifiedJobSerializer):
 
+    result_stdout = serializers.SerializerMethodField()
+
     class Meta:
         model = SystemJob
-        fields = ('*', 'system_job_template', 'job_type', 'extra_vars')
+        fields = ('*', 'system_job_template', 'job_type', 'extra_vars', 'result_stdout')
```

Comment:
The current system job UI looks at this key _instead_ of `/api/v2/system_jobs/N/stdout/`, so we probably have to leave this in for awhile.